frontend/src/@types/product.d.ts

```diff
@@ -0,0 +1,44 @@
+import CommonType from './common';
+
+// TODO: Add rest of fields
+
+export type Product = CommonType & {
+  id?: number;
+  name: string;
+  price: number;
+  dose: string;
+  iva: number;
+  ieps: number;
+  more_info: string;
+  is_generic: string;
+  provider?: number;
+  key: string;
+  status: string;
+  reject_reason: string;
+};
+
+export type CreateProductForm = {
+  name: string;
+  price: number;
+  dose: string;
+  iva: number;
+  ieps: number;
+  more_info: string;
+  is_generic: string;
+  provider?: number;
+  key: string;
+};
```

Comment:
Los errores en los archivos de tipos seguro eran porque, si queremos crear una estructura tipada, se recomienda usar `interface` en lugar de `type`:
```suggestion
export interface CreateProductForm {
  name: string;
  price: number;
  dose: string;
  iva: number;
  ieps: number;
  more_info: string;
  is_generic: string;
  provider?: number;
  key: string;
};
```

Reply:
Debe ser por eso entonces. En el proximo PR intentare volver a las reglas de AirBnb cambiado eso para ver si funciona y arreglar cualquier error de linting nuevo.